pages/documentation.html

```diff
@@ -30,50 +37,150 @@
 
 </div>
 
+<div class="framework">
+<div class="frameworklabel">Books and long tutorials</div>
+<div class="frameworkcontent">
 
+<p>The following books and long tutorials were written by experimented
+Coq users and teachers:</p>
 
-<div class="framework">
-<div class="frameworklabel">Other documentation</div>
+<ul>
 
-<div class="frameworkcontent">
-<p>Moreover, some users of Coq have published various documents that may be useful both to beginners and advanced users of Coq:</p> 
+<li>the <a href="http://www.labri.fr/perso/casteran/CoqArt/">Coq'Art</a>,
```

Comment:
I would remove "the", as in:
```suggestion
<li><a href="http://www.labri.fr/perso/casteran/CoqArt/">Coq'Art</a>,
```
It may also be worth linking to the [Springer Link page](https://link.springer.com/book/10.1007/978-3-662-07964-5) for the book, since some academics with institutional Springer subscriptions can actually download the book directly from there as PDF without cost.